sys-boot/grub/grub-2.12-r5.ebuild

```diff
@@ -368,6 +368,11 @@ src_install() {
 	insinto /etc/default
 	newins "${FILESDIR}"/grub.default-4 grub
 
+	if use uki; then
+		exeinto /etc/grub.d
+		newexe "${FILESDIR}/05_uki-1" 05_uki
+	fi
```

Comment:
Can't we just install this file unconditionally? Do we really need a new USE flag?